pkg/discovery/configs/target_config.go

```diff
@@ -4,8 +4,10 @@ import "errors"
 
 const (
 	// When user doesn't specify username and password, the default username and password will be used.
-	defaultUsername = "defaultUser"
-	defaultPassword = "defaultPassword"
+	defaultUsername      = "defaultUser"
+	defaultPassword      = "defaultPassword"
+	defaultProbeCategory = "CloudNative"
```

Comment:
same category/type will cause problem when adding more than one targets.

Reply:
Discussed with Songbin, for more than one kubernetes clusters, the communication to OpsMgr will fail in one of them. One workaround here is to make catergory/type different for each cluster by appending some random strings.

Reply:
Yes, Songbin and I discussed about it before. We don't have much choice here. The only thing we can improve is to make it less "random". Let's check together